ticketbuyer/buyer.go

```diff
@@ -338,6 +341,21 @@ func (t *TicketPurchaser) SetExpiryDelta(expiryDelta int) {
 	t.purchaserMtx.Unlock()
 }
 
+// SplitTx returns the splittx config value
+func (t *TicketPurchaser) SplitTx() uint32 {
+	t.purchaserMtx.Lock()
+	splitTx := t.splitTx
```

Comment:
Although there is the use of the mutex to protect the use and retrieval of the `splitTx` value, during it's structure in the struct, although not a bad thing, but idiomatically, it is advised to lay within/below a mutex items it would protect, to allow close-ness in memory layout. 

Also, although not necessary to change, consider using `atomic.(Load|Store)Uint32` in `https://golang.org/pkg/sync/atomic` for setting and loading value of `SplitTx` safely and concurrently without the use of mutex.

Reply:
this is in conformity with the paradigms used across the entire dcrwallet project.

Reply:
I agree with @influx6, this fits using `atomic.(Load|Store)Uint32`.